traits/ingresstrait/config/samples/deployment/sample_trait_definition.yaml

```diff
@@ -4,4 +4,5 @@ metadata:
   name: ingresstraits.core.oam.dev
 spec:
   definitionRef:
-    name: ingresstraits.core.oam.dev
\ No newline at end of file
+    name: ingresstraits.core.oam.dev
+  workloadRefPath: spec.workloadRef
```

Comment:
Does this mean `workloadRefPath` is a required field now? What's the rule here?

Reply:
It's not required.  All the catalog trait CRDs ask for that though so we need to supply that.